.github/workflows/nodejs.yml

```diff
@@ -37,7 +37,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: [16.x, 18.x, 20.x]
+        node-version: [18.x, 20.x, 22.x]
```

Comment:
Are we going to skip node 16? If so, I will need to change upstream, it is using node v16.14.0 to build.

Reply:
Yes.